src/Servers/Kestrel/Core/src/Internal/Http2/Http2Connection.cs

```diff
@@ -1192,7 +1192,7 @@ private void StartStream()
         }
 
         KestrelEventSource.Log.RequestQueuedStart(_currentHeadersStream, AspNetCore.Http.HttpProtocol.Http2);
-        _context.ServiceContext.Metrics.RequestQueuedStart(_metricsContext, AspNetCore.Http.HttpProtocol.Http2);
```

Comment:
I find this a little hard to read - it's not really clear from context that that's supposed to be an HTTP version.  Depending on the name of the corresponding parameter, using a named argument might help?  Failing that, a constant could have a name, "Http2MetricName" or something?

Reply:
I added constants